### PR TITLE
SER-3317 | added sleep before checking if user was removed

### PR DESCRIPTION
**Pull request by nlesniewski**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java [hunk 1]
```diff
@@ -170,6 +170,12 @@ public void anonymizationMessageWallRemoved() {
 
     Assertion.assertStringContains(anonymizationPage.getAnonConfirmation(), qanon);
 
+    try {
+      Thread.sleep(5000);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java]: MAJOR Remove this use of "Thread.sleep()". [rule](http://sonarqube.sjc-dev.k8s.wikia.net/coding_rules#rule_key=squid%3AS2925)

[project-id]: com.wikia.selenium-tests

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[project-id]: com.wikia.selenium-tests
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java (hunk 1 above)
Comment (emadry) [reply src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java]: Why sleep instead of waiting for something?

Comment (nlesniewski) [reply src/test/java/com/wikia/webdriver/testcases/desktop/anonymization/AnonymizationTests.java]: Dirty tactical fix. That test already sleeps in other place and needs general refactor.

**Pull request merged**